daml/Marketplace/Investor.daml

```diff
@@ -58,6 +59,28 @@ template Investor
            assert $ deposit.account.owner == investor
            create DepositTransferRequest with sender = investor, senderAccountId = deposit.account.id, ..
 
+      nonconsuming Investor_RequestDeposit : ContractId DepositCreditRequest
```

Comment:
Do these deposit credits get auto-approved by the custodian trigger? 

I'm thinking that they _should_ be auto-approved, at least for the time being, for demo-ing usability. However eventually the custodian will need to verify that a deposit is legit somehow (I'm guessing we'll need DABL integrations with external payment/bank transfer systems?)

Reply:
Yeah it is currently auto-approved by the custodian trigger. It uses `Transfer.DepositDebitRequest` and `Transfer.DepositCreditRequest` which are shared by the exchange ordering system (which I'm pretty sure will always need them to be auto-approved), so if we need to change it so one auto-approves but not the other, we'll probably have to create a specific withdraw/deposit request contract (probably in the custodian module). I'll have to check with Eric if for demo purposes we do want them to be auto-approved.

I'd agree that payment/bank integrations would probably be the best (and niftiest!) way of handling this in the future